2024-11-27 00:46:56,583 INFO    Thread-2  :385645 [wandb_setup.py:_flush():77] Current SDK version is 0.18.1
2024-11-27 00:46:56,584 INFO    Thread-2  :385645 [wandb_setup.py:_flush():77] Configure stats pid to 385645
2024-11-27 00:46:56,584 INFO    Thread-2  :385645 [wandb_setup.py:_flush():77] Loading settings from /home/sjl356/.config/wandb/settings
2024-11-27 00:46:56,584 INFO    Thread-2  :385645 [wandb_setup.py:_flush():77] Loading settings from /home/sjl356/badrobotsIRL/training/binary_models/wandb/settings
2024-11-27 00:46:56,584 INFO    Thread-2  :385645 [wandb_setup.py:_flush():77] Loading settings from environment variables: {'api_key': '***REDACTED***', 'project': 'gru_all_v0', 'entity': 'shannon-jliu', 'root_dir': '/home/sjl356/badrobotsIRL/training/binary_models', 'sweep_id': 'onseyzv7', 'run_id': '5lpm9usw', 'sweep_param_path': '/home/sjl356/badrobotsIRL/training/binary_models/wandb/sweep-onseyzv7/config-5lpm9usw.yaml'}
2024-11-27 00:46:56,584 INFO    Thread-2  :385645 [wandb_setup.py:_flush():77] Applying setup settings: {'mode': None, '_disable_service': None}
2024-11-27 00:46:56,584 INFO    Thread-2  :385645 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program_relpath': 'training/binary_models/gru_model_wandb.py', 'program_abspath': '/home/sjl356/badrobotsIRL/training/binary_models/gru_model_wandb.py', 'program': '/home/sjl356/badrobotsIRL/training/binary_models/gru_model_wandb.py'}
2024-11-27 00:46:56,584 INFO    Thread-2  :385645 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-11-27 00:46:56,585 INFO    Thread-2  :385645 [wandb_init.py:_log_setup():532] Logging user logs to /home/sjl356/badrobotsIRL/training/binary_models/wandb/run-20241127_004656-5lpm9usw/logs/debug.log
2024-11-27 00:46:56,585 INFO    Thread-2  :385645 [wandb_init.py:_log_setup():533] Logging internal logs to /home/sjl356/badrobotsIRL/training/binary_models/wandb/run-20241127_004656-5lpm9usw/logs/debug-internal.log
2024-11-27 00:46:56,585 INFO    Thread-2  :385645 [wandb_init.py:init():616] calling init triggers
2024-11-27 00:46:56,585 INFO    Thread-2  :385645 [wandb_init.py:init():623] wandb.init called with sweep_config: {'activation_function': 'sigmoid', 'batch_size': 32, 'dense_units': 64, 'dropout_rate': 0.8, 'epochs': 500, 'fusion_type': 'early', 'gru_units': 256, 'learning_rate': 0.001, 'loss': 'categorical_crossentropy', 'modality': 'pose', 'num_gru_layers': 1, 'optimizer': 'sgd', 'recurrent_regularizer': 'l1_l2', 'sequence_length': 60, 'use_bidirectional': True, 'use_norm': True, 'use_pca': True, 'use_stats': False}
config: {}
2024-11-27 00:46:56,585 INFO    Thread-2  :385645 [wandb_init.py:init():666] starting backend
2024-11-27 00:46:56,585 INFO    Thread-2  :385645 [wandb_init.py:init():670] setting up manager
2024-11-27 00:46:56,604 INFO    Thread-2  :385645 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-27 00:46:56,613 INFO    Thread-2  :385645 [wandb_init.py:init():678] backend started and connected
2024-11-27 00:46:56,645 INFO    Thread-2  :385645 [wandb_run.py:_config_callback():1388] config_cb None None {'activation_function': 'sigmoid', 'batch_size': 32, 'dense_units': 64, 'dropout_rate': 0.8, 'epochs': 500, 'fusion_type': 'early', 'gru_units': 256, 'learning_rate': 0.001, 'loss': 'categorical_crossentropy', 'modality': 'pose', 'num_gru_layers': 1, 'optimizer': 'sgd', 'recurrent_regularizer': 'l1_l2', 'sequence_length': 60, 'use_bidirectional': True, 'use_norm': True, 'use_pca': True, 'use_stats': False}
2024-11-27 00:46:56,653 INFO    Thread-2  :385645 [wandb_init.py:init():773] updated telemetry
2024-11-27 00:46:56,704 INFO    Thread-2  :385645 [wandb_init.py:init():806] communicating run to backend with 90.0 second timeout
2024-11-27 00:46:56,893 INFO    Thread-2  :385645 [wandb_init.py:init():857] starting run threads in backend
2024-11-27 00:46:57,314 INFO    Thread-2  :385645 [wandb_run.py:_console_start():2459] atexit reg
2024-11-27 00:46:57,315 INFO    Thread-2  :385645 [wandb_run.py:_redirect():2307] redirect: wrap_raw
2024-11-27 00:46:57,316 INFO    Thread-2  :385645 [wandb_run.py:_redirect():2372] Wrapping output streams.
2024-11-27 00:46:57,317 INFO    Thread-2  :385645 [wandb_run.py:_redirect():2397] Redirects installed.
2024-11-27 00:46:57,327 INFO    Thread-2  :385645 [wandb_init.py:init():900] run started, returning control to user process
2024-11-27 00:54:44,253 ERROR   Thread-1  :385645 [internal_api.py:execute():349] 502 response executing GraphQL.
2024-11-27 00:54:44,254 ERROR   Thread-1  :385645 [internal_api.py:execute():350] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2024-11-27 11:09:13,303 ERROR   Thread-1  :385645 [internal_api.py:execute():349] 502 response executing GraphQL.
2024-11-27 11:09:13,304 ERROR   Thread-1  :385645 [internal_api.py:execute():350] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2024-11-27 11:13:47,178 ERROR   Thread-1  :385645 [internal_api.py:execute():349] 502 response executing GraphQL.
2024-11-27 11:13:47,179 ERROR   Thread-1  :385645 [internal_api.py:execute():350] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2024-11-27 13:27:49,947 ERROR   Thread-1  :385645 [internal_api.py:execute():349] 502 response executing GraphQL.
2024-11-27 13:27:49,948 ERROR   Thread-1  :385645 [internal_api.py:execute():350] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2024-11-27 13:29:16,546 ERROR   Thread-1  :385645 [internal_api.py:execute():349] 502 response executing GraphQL.
2024-11-27 13:29:16,547 ERROR   Thread-1  :385645 [internal_api.py:execute():350] 
<html><head>
<meta http-equiv="content-type" content="text/html;charset=utf-8">
<title>502 Server Error</title>
</head>
<body text=#000000 bgcolor=#ffffff>
<h1>Error: Server Error</h1>
<h2>The server encountered a temporary error and could not complete your request.<p>Please try again in 30 seconds.</h2>
<h2></h2>
</body></html>

2024-11-27 13:32:30,339 INFO    Thread-2  :385645 [wandb_run.py:_finish():2158] finishing run shannon-jliu/gru_all_v0/5lpm9usw
2024-11-27 13:32:30,340 INFO    Thread-2  :385645 [wandb_run.py:_atexit_cleanup():2422] got exitcode: 0
2024-11-27 13:32:30,340 INFO    Thread-2  :385645 [wandb_run.py:_restore():2404] restore
2024-11-27 13:32:30,340 INFO    Thread-2  :385645 [wandb_run.py:_restore():2410] restore done
2024-11-27 13:32:32,125 INFO    Thread-2  :385645 [wandb_run.py:_footer_history_summary_info():4037] rendering history
2024-11-27 13:32:32,128 INFO    Thread-2  :385645 [wandb_run.py:_footer_history_summary_info():4069] rendering summary
2024-11-27 13:32:32,149 INFO    Thread-2  :385645 [wandb_run.py:_footer_sync_info():3996] logging synced files
